From 525f2307fa3e2d0ae55c8c922e6f7220a1e5bd1b Mon Sep 17 00:00:00 2001
From: Mark Reynolds <mreynolds@redhat.com>
Date: Thu, 3 Feb 2022 16:51:38 -0500
Subject: [PATCH] Issue 4775 - Fix cherry-pick error

Bug Description: EntryUUID when added was missing it's CLI
and helpers for fixups.

Fix Description: Add the CLI elements.

fixes: https://github.com/389ds/389-ds-base/issues/4775

Author: William Brown <william@blackhats.net.au>

Review by: @mreynolds389 (thanks!)
---
 src/lib389/lib389/cli_conf/plugin.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/lib389/lib389/cli_conf/plugin.py b/src/lib389/lib389/cli_conf/plugin.py
index 7c0cf2c80..fb0ef3077 100644
--- a/src/lib389/lib389/cli_conf/plugin.py
+++ b/src/lib389/lib389/cli_conf/plugin.py
@@ -27,7 +27,6 @@ from lib389.cli_conf.plugins import passthroughauth as cli_passthroughauth
 from lib389.cli_conf.plugins import retrochangelog as cli_retrochangelog
 from lib389.cli_conf.plugins import automember as cli_automember
 from lib389.cli_conf.plugins import posix_winsync as cli_posix_winsync
-from lib389.cli_conf.plugins import contentsync as cli_contentsync
 from lib389.cli_conf.plugins import entryuuid as cli_entryuuid
 
 SINGULAR = Plugin
@@ -115,7 +114,6 @@ def create_parser(subparsers):
     cli_passthroughauth.create_parser(subcommands)
     cli_retrochangelog.create_parser(subcommands)
     cli_posix_winsync.create_parser(subcommands)
-    cli_contentsync.create_parser(subcommands)
     cli_entryuuid.create_parser(subcommands)
 
     list_parser = subcommands.add_parser('list', help="List current configured (enabled and disabled) plugins")
-- 
2.34.1

